src/entity/user.ts

```diff
@@ -0,0 +1,10 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+@Entity()
+export class User {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  Name: string;
```

Comment:
o `id` está começando com minúsculo mas `Name` não, cuidado com a consistência
```suggestion
  name: string;
```